open-sphere-base/core/src/main/java/io/opensphere/core/util/fx/tabpane/ElementsMapElement.java

```diff
@@ -0,0 +1,50 @@
+package io.opensphere.core.util.fx.tabpane;
+
+import javafx.beans.InvalidationListener;
+
+/** A simple class in which an element's listener and counter are defined. */
+public class ElementsMapElement
+{
+    /** The listener for the element. */
+    private InvalidationListener myListener;
+
+    /** The counter for the element. */
+    private int myCounter;
+
+    /**
+     * Creates a new element with the supplied invalidation listener.
+     *
+     * @param listener the listener for the element's invalidation.
+     */
+    public ElementsMapElement(InvalidationListener listener)
+    {
+        this.myListener = listener;
```

Comment:
I think some of your eclipse save actions aren't getting run, like removing "this", adding final to member variables.

Reply:
Fixed